dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/alias/UrlAliasRepositoryImpl.java

```diff
@@ -0,0 +1,289 @@
+                      .stream()
+                      .toArray(UrlAlias[]::new));
+      return new SearchPageResponse<>(
+          List.of(new LocalizedUrlAliases(resultset)), searchPageRequest, count);
+    } catch (StatementException e) {
+      String detailMessage = e.getCause() != null ? e.getCause().getMessage() : e.getMessage();
+      throw new UrlAliasRepositoryException(
+          String.format("The SQL statement is defective: %s", detailMessage), e);
+    } catch (JdbiException e) {
+      throw new UrlAliasRepositoryException(e);
+    }
+  }
+
+  @Override
+  public LocalizedUrlAliases findAllForTarget(UUID uuid) throws UrlAliasRepositoryException {
+    if (uuid == null) {
+      return new LocalizedUrlAliases();
+    }
+    String sql =
+        "SELECT * FROM " + TABLE_NAME + " WHERE target_uuid = ? ORDER BY main_alias, slug;";
+    try {
+      UrlAlias[] resultset =
+          this.dbi.withHandle(
+              h ->
+                  h.createQuery(sql).bind(0, uuid).mapToBean(UrlAlias.class).stream()
+                      .toArray(UrlAlias[]::new));
+      return new LocalizedUrlAliases(resultset);
+    } catch (StatementException e) {
+      String detailMessage = e.getCause() != null ? e.getCause().getMessage() : e.getMessage();
+      throw new UrlAliasRepositoryException(
+          String.format("The SQL statement is defective: %s", detailMessage), e);
+    } catch (JdbiException e) {
+      throw new UrlAliasRepositoryException(e);
+    }
+  }
+
+  @Override
+  public LocalizedUrlAliases findMainLinks(UUID websiteUuid, String slug)
+      throws UrlAliasRepositoryException {
+    if (websiteUuid == null || !StringUtils.hasText(slug)) {
+      return new LocalizedUrlAliases();
+    }
+    String innerSel =
+        "(SELECT target_uuid FROM " + TABLE_NAME + " WHERE website_uuid = :uuid AND slug = :slug)";
+    String sql =
+        "SELECT * FROM "
+            + TABLE_NAME
+            + " WHERE website_uuid = :uuid AND main_alias = true AND target_uuid = "
+            + innerSel;
+    Map<String, Object> bindings = Map.of("uuid", websiteUuid, "slug", slug);
+    try {
+      UrlAlias[] resultset =
+          this.dbi.withHandle(
+              h ->
+                  h.createQuery(sql).bindMap(bindings).mapToBean(UrlAlias.class).stream()
+                      .toArray(UrlAlias[]::new));
+      return new LocalizedUrlAliases(resultset);
+    } catch (StatementException e) {
+      String detailMessage = e.getCause() != null ? e.getCause().getMessage() : e.getMessage();
+      throw new UrlAliasRepositoryException(
+          String.format("The SQL statement is defective: %s", detailMessage), e);
+    } catch (JdbiException e) {
+      throw new UrlAliasRepositoryException(e);
+    }
+  }
+
+  @Override
+  public UrlAlias findOne(UUID uuid) throws UrlAliasRepositoryException {
+    if (uuid == null) {
+      return null;
+    }
+    String sql = "select * from " + TABLE_NAME + " where uuid = ?;";
```

Comment:
```suggestion
    String sql = "select * from " + TABLE_NAME + " where uuid = :uuid;";
```
Damit kannst du unten beim `bind` den Namen statt `0` angeben.